constants.js

```diff
@@ -0,0 +1,28 @@
+const SKUS = {
+  'MKGR3LL/A': '14" Si, Base',
+  'MKGP3LL/A': '14" SG, Base',
+  'MKGT3LL/A': '14" Si, Better',
+  'MKGQ3LL/A': '14" SG, Better',
+  'MMQX3LL/A': '14" Si, Ultimate',
+  'MKH53LL/A': '14" SG, Ultimate',
+  'MK1E3LL/A': '16" Si, Base',
+  'MK183LL/A': '16" SG, Base',
+  'MK1F3LL/A': '16" Si, Better',
+  'MK193LL/A': '16" SG, Better',
+  'MK1H3LL/A': '16" Si, Best',
+  'MK1A3LL/A': '16" SG, Best',
+  'MK233LL/A': '16" SG, Ultimate',
+  'MMQW3LL/A': '16" Si, Ultimate',
+  'MYD92LL/A': '13" Control',
+};
+
+const COUNTRIES = {
+  US: "",
+  CA: "/ca",
+  AU: "/au", // Issue with fetching australian stock
```

Comment:
I saw the same issue with fetching Australian data - not sure what's going on there. Browsing around a bit on the `au` Apple Store, my best guess is that some fundamental difference with in-store pickup in Australia vs North America. weird.